README.md

```diff
@@ -1 +1,57 @@
-# To-Do-list
\ No newline at end of file
+# To Do list
+
+Live preview [here](https://vudej.github.io/To-Do-list/). 
+
+## Screenshots
+![Example screenshot](img/screenshot.PNG)
+
+## Contents
+* [Homepage](#homepage)
+
```

Comment:
- [ ] **Required:** Your live demo link opens to your Readme. The link isn't right. Please confirm the link.